QuickDialog.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'QuickDialog'
-  s.version  = '1.0.2'
+  s.version  = '1.1.0'
```

Comment:
I think this would be a good patch level, but since we are diverging so much I am fine with a minor version.